packages/components/src/styles/table-cell/table-cell.stories.ts

```diff
@@ -41,580 +44,216 @@ export const Default = {
-                    id="sortableHeader-${columnIndex}"
-                    aria-sort=${ifDefined(sortData[columnIndex] === 'none' ? undefined : 'ascending')}
-                  >
-                    <button
-                      class="text-primary hover:bg-neutral-200 transition-all"
-                      @click="${() => sortTable(columnIndex)}"
-                    >
-                      ${cellData}<sd-icon
-                        id="sortIcon-${columnIndex}"
-                        library="global-resources"
-                        name=${sortData[columnIndex] === 'none' ? 'system/sort-down' : 'system/sort-down-filled'}
-                        class="text-[12px]"
-                      ></sd-icon>
-                    </button>
-                  </th>`;
-                })}
-              </tr>`;
-            })()}
-          </thead>
+        <table class="sd-table">
           <tbody>
-            ${tableData.map(rowData => {
-              return html`<tr>
-                ${rowData.map(cellData => {
-                  return html`<td class="sd-table-cell sd-table-cell--bg-transparent">
-                    ${Math.floor(Math.random() * 10)}: ${cellData}
-                  </td>`;
-                })}
-              </tr>`;
-            })}
+            <tr class="relative">
+              <td class="sd-table-cell sd-table-cell--bg-neutral-100">Neutral-100</td>
+            </tr>
           </tbody>
         </table>
-
-        <div class="headline">Multi Select Table</div>
-        <div class="disclaimer">This sample will be provided soon.</div>
       </div>
     `;
   }
 };
 
 /**
- * These examples demonstrate the usage of sd-table-cell when the header is sticky and a shadow is displayed.
- *
- * ```html
- * <table id="vertical-scrollable-table" class="sd-table block overflow-y-scroll">
- *   <thead>
- *     <tr>
- *       <th class="sd-table-cell sd-table-cell--bg-white sticky top-0 sd-table-cell--shadow-bottom">
- *         Header
- *       </th>;
- *     </tr>;
- *   </thead>
- *   <tbody>
- *     <tr>
- *       <td class="sd-table-cell sd-table-cell--bg-transparent">
- *         Lorem ipsum
- *       </td>
- *     </tr>
- *   </tbody>
- * </table>
- * ```
- *
- * To make the bottom shadow visible, the following JavaScript code is required:
- *
- * ```js
- * document.addEventListener('DOMContentLoaded', () => {
- *   const scrollableTable = document.getElementById('vertical-scrollable-table');
- *   const tableHeaders = scrollableTable.querySelectorAll('th');
- *
- *   scrollableTable.addEventListener('scroll', () => {
- *     tableHeaders.forEach(header => {
- *       if (scrollableTable.scrollTop === 0) {
- *         header.classList.remove('sd-table-cell--shadow-active');
- *       } else {
- *         header.classList.add('sd-table-cell--shadow-active');
- *       }
- *     });
- *   });
- * });
- * ```
+ * Use the `&--divider` class to toggle vertical dividers.
```

Comment:
for the "Divider" sample: can you make it more similar to the Figma sample? size wise 325 weight

Reply:
Updated